source/client.lua

```diff
@@ -197,9 +208,29 @@ for i=1, #Config do
     createClothingStore(Config[i])
 end
 
+
+
+
+
+
 AddEventHandler("onResourceStop", function(resource)
     if resource ~= cache.resource then return end
     SetResourceKvp(wardrobeId, json.encode(wardrobe))
+    SetResourceKvp("ND_AppearanceShops:savedclothing", json.encode(Outfit))
+    for _, ped in pairs(Peds) do
+        NDCore.removeAiPed(ped)

```

Comment:
ND Core should do this.

Reply:
None of the stuff you say it should do happens for us. These updates were so that when we changed our appearances, we would have them when we loaded back in. Which Never Happened

Reply:
It makes it so that the last appearance you saved in an Appearance shop loads in instead of the character you can only set once.